Assets/PatchKit Patcher/Scripts/AppData/Local/TemporaryDirectory.cs

```diff
@@ -13,14 +14,21 @@ public sealed class TemporaryDirectory : IDisposable
     {
         public string Path { get; private set; }
 
-        public TemporaryDirectory([NotNull] string path)
+        private bool _keep = false;
+
+        private TemporaryDirectory([NotNull] string path)
         {
             if (string.IsNullOrEmpty(path))
             {
                 throw new ArgumentException("Value cannot be null or empty.", "path");
             }
 
             Path = path;
+            
+            if (Directory.Exists(Path))
+            {
+                Directory.Delete(Path);
```

Comment:
This will throw exception if directory is not empty. You need to do `Directory.Delete(Path, true)`.